params/config.go

```diff
@@ -256,6 +256,7 @@ var (
 
 	// ScrollAlphaChainConfig contains the chain parameters to run a node on the Scroll Alpha test network.
 	ScrollFeeVaultAddress           = common.HexToAddress("0x5300000000000000000000000000000000000005")
```

Comment:
we should move this to rollup/rcfg/config.go

Reply:
Agreed, we can do it in another PR.